pkg/workloadmeta/collectors/internal/remote/workloadmeta/workloadmeta_test.go

```diff
@@ -171,6 +172,11 @@ func TestCollection(t *testing.T) {
 
 	mockClientStore := workloadmeta.NewMockStore()
 	ctx, cancel := context.WithCancel(context.TODO())
+	// Subscribe to the mockStore and wait for the initial event
+	ch := mockClientStore.Subscribe("dummy", workloadmeta.NormalPriority, nil)
+	<-ch
```

Comment:
Can this read on `ch` channel be removed now that the number of events is checked in the `for` loop line 231 ?

```suggestion
```

Reply:
Indeed I forgot it 😅